triangle.js

```diff
@@ -0,0 +1,66 @@
+class isTriangle {
```

Comment:
it might make more sense to have this be named `Triangle` - classes are generally capitalized as well